DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -1457,6 +1462,26 @@ public async Task<IReadOnlyList<DiscordAuditLogEntry>> GetAuditLogsAsync(int? li
                 ahd = amh.ToDictionary(xh => xh.Id, xh => xh);
             }
 
+            Dictionary<ulong, DiscordScheduledGuildEvent> events = null;
+            if (eve.Any())
+            {
+                var evh = await this.GetEventsAsync().ConfigureAwait(false);
```

Comment:
There's a comment above about how the event will only have the ID if it's not cached, yet there's an API call here (which you should generally avoid unless explicitly stated because nobody likes incurring random ratelimits). So, which is it?

Reply:
I'll switch it to cached only